tools/generator/src/Generator/Generator.swift

```diff
@@ -194,6 +195,7 @@ class Generator {
             files,
             internalDirectoryName,
             bazelIntegrationDirectory,
+            swiftcStub,
```

Comment:
Instead of this, let's just copy it in the installer like we do for the bazelrc path.

I'll make a future change that optimizing the setup of that directory, taking it out of the generator entirely, leveraging more of bazel's caching.

Reply:
Fixed in https://github.com/buildbuddy-io/rules_xcodeproj/pull/1198/commits/031500f2b980a6c25f3391cae3fcf3080ce59445.